content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -102,18 +102,16 @@ By default, a waypoint will only handle traffic destined for **services** in its
 
 It is also possible for the waypoint to handle all traffic, only handle traffic sent directly to **workloads** (pods or VMs) in the cluster, or no traffic at all. The types of traffic that will be redirected to the waypoint are determined by the `istio.io/waypoint-for` label on the `Gateway` object.
 
-The `--for` parameter to `istioctl waypoint apply` can be used to change the [traffic type](#waypoint-traffic-types) redirected to the waypoint:
+Use the `--for` flag on `istioctl waypoint apply` to change the types of traffic that can be redirected to the waypoint:
 
-| `waypoint-for` value | Traffic type |
+| `waypoint-for` value | Original Destination Type |
 | -------------------- | ------------ |
 | `service`            | Kubernetes services |
 | `workload`           | Pod IPs or VM IPs |
 | `all`                | Both service and workload traffic |
 | `none`               | No traffic (useful for testing) |
 
-{{< tip >}}
-Traffic type concerns the original addressing. To-service traffic does not become to-workload once the service is resolved to a pod by the service mesh.
-{{< /tip >}}
+Waypoint selection occurs based on the type of destination, `service` or `workload`, which traffic was **originally** addressed to. If traffic is addressed to a service which does not have a waypoint then waypoint will not be transitted even if the eventual workload it reaches does have an attached waypoint. 
```

Comment:
```suggestion
Waypoint selection occurs based on the type of destination, `service` or `workload`, which traffic was **originally** addressed to. If traffic is addressed to a service which does not have a waypoint then waypoint will not be transited even if the eventual workload it reaches does have an attached waypoint. 
```

Reply:
(ty)

Reply:
ty, for some reason my local lsp for markdown (which checks spelling) hates these hugo not quite markdown files...